nifi-iotics-hostservice/pom.xml

```diff
@@ -21,6 +21,7 @@
         <groupId>smartrics.iotics.nifi</groupId>
         <artifactId>nifi-iotics</artifactId>
         <version>1.1-SNAPSHOT</version>
+        <relativePath>..</relativePath>
```

Comment:
isn't this the default value? why do we need this

Reply:
the reason I added it was that I was trying to build just the processor project and it couldn't find some of the dependencies.
I'm not sure it's defaulted, as I remember if you don't set it , it takes it from the local maven repo instead of the one above. (I could be wrong though)